samples/minimum_criteria.rst

```diff
@@ -0,0 +1,20 @@
+.. _definition_and_minimum_criteria:
+
+Definition & Minimum Criteria
+#############################
+
+A sample is an application in Zephyr that briefly showcases how to use a feature, subsystem, or domain.
+
+===========================================
+Minimum Requirements/ Criteria for Samples:
+===========================================
+
+* Samples must not use Zephyr's testing framework.
+* Leveraged in CI for quality confidence, its primary purpose is to exemplify the usage of subsystem, feature, or domain.
```

Comment:
Not exactly sure what that means and how samples contribute to quality in CI

Reply:
@nashif How about this reframed point? 

"Samples must not be used as a test case, even though they are executed in CI as a pre-verification step before a pull request gets merged."